roles/edpm_pre_adoption_validation/tasks/hostname.yml

```diff
@@ -25,49 +25,52 @@
         path: "{{ edpm_pre_adoption_validation_old_nova_config }}"
       register: nova_old_config_file
 
-    # NOTE(gibi): we intentionally using python configparser to extract
-    # the config value. Grep is not enough as multiple sections have 'host'
-    # key. Also we cannot rely on crudini as it is not available on the
-    # node being adopted.
-    - name: Compare old nova service host to {{ canonical_hostname }}
+    - name: Retrieve original hostname
       when: nova_old_config_file.stat.exists
-      ansible.builtin.command:
-        cmd: >
-          python3 -c
-          "import configparser as c;
```

Comment:
we were usign configparser before because the specific files we are interested in is not using the oslo extensions to the ini format so it can be read with any ini parser such as configparser.